solutions/project/hub-env/setters.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # Billing Account ID to be associated with this project
+  project-billing-id: "AAAAAA-BBBBBB-CCCCCC"
+  
+  # GCP folder to use as parent to this project, lowercase K8S resource name
+  project-parent-folder: project-parent-folder
+  
+  # Naming Convention for project-id : <tenant-code><environment-code>m<data-classification>-<project-owner>-<user defined string>
+  # Max 30 characters
+  hub-project-id: xxdmu-admin1-projectname
+
+  # Identity that should be allowed to access the management VM using IAP TCP forwarding
+  # https://cloud.google.com/iap/docs/using-tcp-forwarding
+  hub-admin: group:group@domain.com
+
+  #################
+  # Org Policies
+  #######
+  # This list constraint defines the set of VPC networks
+  # that are allowed to be peered with the VPC networks belonging to this project, see YAML file for more info:
+  # org-policies/exceptions/compute-restrict-vpc-peering-except-hub-project.yaml
+  # this setting MUST be changed to include the ORG ID
+  project-allowed-restrict-vpc-peering: |
+    - under:organizations/ORGANIZATION_ID
+  # This list constraint defines the set of Compute Engine VM instances that are allowed to use external IP addresses, see YAML file for more info:
+  # org-policies/exceptions/compute-vm-external-ip-access-except-hub-project.yaml
+  # this setting MUST be changed to include the hub project ID
+  project-allowed-vm-external-ip-access: |
+    - "projects/PERIMETER_PROJECT_ID/zones/northamerica-northeast1-a/instances/fgt-primary-instance"
```

Comment:
maybe `PERIMETER_PROJECT_ID` should be renamed to `HUB_PROJECT_ID` to avoid confusion?

Reply:
good catch... renaming PERIMETER_PROJECT_ID to HUB_PROJECT_ID was one of my latest change...I missed that one